sycl/include/sycl/queue.hpp

```diff
@@ -21,6 +21,7 @@
 #include <sycl/detail/id_queries_fit_in_int.hpp> // for checkValueRange
 #include <sycl/detail/info_desc_helpers.hpp>  // for is_queue_info_...
 #include <sycl/detail/kernel_desc.hpp>        // for KernelInfo
+#include <sycl/detail/nd_range_view.hpp>      // for nd_range_view
```

Comment:
Let's remove the comment because it is useless. The `nd_range_view.hpp` name is self-descriptive.